openant/devices/shift.py

```diff
@@ -28,6 +28,29 @@ class ShiftingSystemID(Enum):
     def _missing_(cls, _):
         return ShiftingSystemID.Unknown
 
+class FunctionSetEventTypes(Enum):
```

Comment:
Pedantic but can you rename to `FunctionSetEventType` since that's how it's used (singular).